benchmark/msm/arkworks/src/lib.rs

```diff
@@ -19,10 +20,12 @@ pub extern "C" fn run_msm_arkworks(
         let mut bases_vec = Vec::<G1Affine>::new();
         bases_vec.reserve_exact(bases.len());
         for i in 0..size {
+            let x = mem::transmute(bases[i].x);
+            let y = mem::transmute(bases[i].y);
             bases_vec.push(G1Affine {
-                x: mem::transmute(bases[i].x),
-                y: mem::transmute(bases[i].y),
-                infinity: bases[i].infinity,
+                x,
+                y,
```

Comment:
I needed to call `is_zero()` method to check whether the point is point at infinity. Since it is defined by arkworks, so I needed to `transmute` to call this method.